src/Mollie/WC/Gateway/Giftcard.php

```diff
@@ -97,7 +97,8 @@ public function payment_fields()
 
 		// If only one gift card issuers is available, show it without a dropdown
 		if ( count( $issuers ) === 1 ) {
-			$html .= '<img src="' . $this->getIssuerIconUrl( $issuers[0]->id ) . '" style="vertical-align:middle" />';
+            $issuerImageSvg = $this->checkSvgIssuers($issuers);
+			$html .= '<img src="' . $issuerImageSvg . '" style="vertical-align:middle" />';
```

Comment:
Image markup should not be added to the output if `$this->checkSvgIssuers($issuers)` returns an empty string.